wgpu-core/src/command/bind.rs

```diff
@@ -219,6 +219,13 @@ impl Binder {
         }
     }
 
+    pub(super) fn list_active<'a>(&'a self) -> impl 'a + Iterator<Item = Valid<BindGroupId>> {
```

Comment:
nit: this could be `fn list_active(&self) -> impl Iterator<Item = Valid<BindGroupId>> + '_`